packages/dx-react-grid/docs/guides/virtual-scrolling.md

```diff
@@ -38,6 +38,16 @@ If the Grid should have the same size as the container element, set the `Virtual
 
 .embedded-demo({ "path": "grid-virtual-scrolling/stretching-to-parent-element", "showThemeSelector": true })
 
+## Scroll to Row
+
+To scroll the table to a particular row, call the `scrollToRow` method. It accepts a row ID. To call the method, you need the `VirtualTable` plugin's ref.
+
+In the following demo, the `scrollToRow` method is used to scroll the table to a new or saved row. When you add a new row, it is added to the table's top, and the table is scrolled to it. When you save the row, its position is changed according to sorting, and the table is scrolled to that position.
+
+.embedded-demo({ "path": "grid-virtual-scrolling/scroll-to-row", "showThemeSelector": true })
+
+NOTE: Scrolling to a row does not work with [lazy loading](./lazy-loading.md/#react-grid---virtual-scrolling-with-remote-data-lazy-loading). This feature requires the IDs of all rows, but in lazy loading mode, the Grid loads rows in parts.
```

Comment:
```suggestion
NOTE: Scrolling to a row cannot be used with [lazy loading](./lazy-loading.md/#react-grid---virtual-scrolling-with-remote-data-lazy-loading). This is because the Grid loads rows in parts in lazy loading mode and this feature requires all the row IDs.
```

Reply:
```suggestion
NOTE: Scrolling to a row cannot be used with [lazy loading](./lazy-loading.md/#react-grid---virtual-scrolling-with-remote-data-lazy-loading). This is because the Grid loads rows in parts in lazy loading mode, and scrolling to a row requires all the row IDs.
```